01week/resume/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Seung Ju Han Resume</title>
+</head>
+<body>
+    <header>
+        <h1>Seung Ju Han</h1>
+        <nav>
+            <a href = "#">LinkedIn</a> <a href = "#">Personal Website</a> <a href ="#">Facebook</a>
```

Comment:
Avoid having spaces between attribute, equal sign, and value.
NO - `href = "#"`
YES - `href="#"`